question09/main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// loading img 
+const wrapper = document.getElementById('wrapper');
+const loadImg = document.createElement('img');
+loadImg.src = "loading-circle.gif";
+wrapper.appendChild(loadImg);
+
+
+const contentsList = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+                      {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}]
+
+
+const promise = new Promise((resolve) => {
+    setTimeout(() =>{
+      resolve(contentsList);
+    }, 3000)
+})
+
+async function asyncFunc() {
+    const result = await promise;
+    createList(result);
+    loadImg.remove();
+}
+asyncFunc();
+
```

Comment:
②The "await" waits for processing until the result of the specified Promise is returned.
③After the resolved promise is returned, the processes in the asyncFunc function will be executed in order.